### ISSUE TITLE
Wallpaper disappears when clicking on desktop (Garuda Kde Wayland)

### ISSUE THREAD

**Issue by davigamer987** [GhostNaN/mpvpaper#13]
Wallpaper disappears when clicking on desktop (Garuda Kde Wayland), When I click on the desktop the wallpaper disappears, When I'm clicking in any other window it works fine

**Comment by GhostNaN** [comment]
I found a similar problem when I tried this myself for fun. It's pretty cool that it works at all on KDE as it's not supported. Sorry that it doesn't fully work on KDE, but I had no intention of this running on KDE's KWin compositor. After all "MPVPaper is a wallpaper program for wlroots based wayland compositors".

But.... that doesn't mean this could never work. Very soon KWinFT (a KWin replacement) will have a wlroots backend:
https://gitlab.com/kwinft/kwinft/-/merge_requests/108

I haven't tested this myself yet, but in theory is should work. We'll see...
